requestor/run_service_manager.py

```diff
@@ -3,26 +3,36 @@
 from service_manager import ServiceManager
 import erigon_services
 
+EXECUTOR_CFG = {
+    'budget': 1,
+    'subnet_tag': 'erigon',
```

Comment:
I would be nice to have subnet tag configurable by command line param and/or env variable.

Reply:
It is configurable in the main script (`server.py`).

This is only dev/demonstration script, I think the more explicit is configuration such places the better.